public/app/plugins/datasource/influxdb/datasource.ts

```diff
@@ -240,7 +240,7 @@ export default class InfluxDatasource extends DataSourceWithBackend<InfluxQuery,
     if (this.isFlux) {
       return {
         ...query,
-        query: this.templateSrv.replace(query.query ?? '', rest), // The raw query text
+        query: this.templateSrv.replace(query.query ?? '', rest, 'regex'), // The raw query text
```

Comment:
So this means default will change to regex right. Wonder could this break anything? Like we do a regex escaping then which probably makes for multiple values and influx but what about single value? Is it possible we will now escape something (as in regex) that should not be escaped and thus changing some currently existing queries?

Reply:
It doesn't break anything. We've checked it together with @yesoreyeram and @joey-grafana. Since this has been done after migration we compared the code that was running smoothly on the frontend side. And then we applied the same here.